docs/sources/shared/configuration.md

```diff
@@ -3765,7 +3765,7 @@ shard_streams:
 [bloom_creation_enabled: <boolean> | default = false]
 
 # Experimental. Bloom planning strategy to use in bloom creation. Can be one of:
-# 'split_keyspace_by_factor'
+# 'split_keyspace_by_factor', 'split_by_chunk_size'
```

Comment:
I'm not entirely happy with this name. Some other options I thought were:
- `split_by_tsdb_chunks_size_stats`
   - This is probably my favorite 
- `split_by_chunks_size_stats`
- `split_by_series_chunks_size`
- `split_by_series_size`

Preferences? Ideas?

Reply:
I like `split_by_tsdb_chunks_size` to make it clear which chunks we're talking about; `split_by_tsdb_chunks_size_stats` is fine too, though I'm not sure if the `_stats` bit adds any more information to make it more clear.

Reply:
I would prefer `split_by_series_size` or `split_by_stream_size`

Reply:
>I would prefer split_by_series_size or split_by_stream_size

I would like to specifically have "chunks" somewhere in the name provided that in the future we may look at other stats besides the chunk sizing. But I may be overthinking it 😅

Reply:
I'm going with `split_by_series_chunks_size`